python/pyhie/allspark/model/deepseek_v3.py

```
@@ -640,6 +712,39 @@ def _trans_weight(self, torch_weight, lora_name=None):
+        def _dequant_fp8_cpu(tensor, scale_inv):
+            """Dequantize FP8 tensor on CPU using block-wise scale_inv."""
+            w = tensor.to(_torch_target_dtype)
+            s = scale_inv.to(_torch_target_dtype)
+            bm, bn = self.fp8_block_size
+            if s.dim() == 2 and w.dim() == 2:
+                M, N = w.shape
+                s_exp = s.repeat_interleave(bm, dim=0).repeat_interleave(bn, dim=1)
+                w = w * s_exp[:M, :N]
+            elif s.dim() == 3 and w.dim() == 3:
+                # Stacked expert weights: [num_experts, K, N]
+                E, M, N = w.shape
```

Ruff (0.15.1)

[warning] Unpacked variable `E` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)